### PR TITLE
M-3 Add health check endpoint

### PR DESCRIPTION
**Pull request by onet1001**
Ready for review

### CODE REVIEW

**Comment by kaburen** [comment]
@onet1001 add labels, connected issue, etc. please

File: src/test/groovy/it/move2/manager/rest/StatusControllerISpec.groovy [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        expect:
+        mvc.perform(get("/health-check"))
+                .andExpect(status().isOk())
+                .andExpect(content().string("OK"))
+    }
+
+    def "Should not return other response"() {
+        expect:
+        mvc.perform(get("/health-check"))
+                .andExpect(status().isOk())
```
[the remark this answers is not in the document]
Comment (Corel-Cormen) [reply src/test/groovy/it/move2/manager/rest/StatusControllerISpec.groovy:35]: Ack

**Review by Corel-Cormen — APPROVED** [review]

**Review by mateuszjanczak — APPROVED** [review]
LGTM

**Review by Corel-Cormen — APPROVED** [review]

**Comment by pikuskrystian** [comment]
LGMT

**Review by pikuskrystian — APPROVED** [review]
LGTM

**Pull request merged**